packages/theme/src/cli/flags.ts

```diff
@@ -13,4 +13,15 @@ export const themeFlags = {
     env: 'SHOPIFY_FLAG_PATH',
     default: '.',
   }),
+  password: Flags.string({
+    hidden: false,
+    description: 'Password generated from the Theme Access app.',
+    env: 'SHOPIFY_CLI_THEME_TOKEN',
+  }),
+  store: Flags.string({
+    char: 's',
+    description: 'Store URL.',
```

Comment:
Is this the URL or the domain? What about including an example to prevent some confusion for the user?

Reply:
Done!